casper/src/main/rholang/MakeMint.rho

```diff
@@ -18,6 +18,32 @@ new
 in {
   rl!(`rho:id:nd74ztexkao5awjhj95e3octkza7tydwiy7euthnyrt5ihgi9rj495`, *NonNegativeNumberCh) |
   for(@(_, NonNegativeNumber) <- NonNegativeNumberCh) {
+    new roguePurse1, trace(`rho:io:stderr`) in {
```

Comment:
please write a Rholang unittest that fails before this change and succeeds after.

If you rebase on top of https://github.com/rchain/rchain/pull/2100 you will be able to use RhoSpec for it and your unittesting experience will be considerably better